source/assets/stylesheets/_accessibility.scss

```diff
@@ -12,9 +12,9 @@
   }
 }
 
-/* Hide for both screenreaders and browsers */
+/* Hide for both screenreaders and browsers when JS enabled */
```

Comment:
This comment only applies to the second selector, is there a better way to comment this?

Reply:
True - we can just lose the JS bit.